docs/src/time_integration.md

```diff
@@ -42,3 +44,170 @@ are the following. Further documentation can be found in the
+In Trixi.jl, the former approach is implemented by means of convex optimization using the [Convex.jl](https://github.com/jump-dev/Convex.jl) package.
+
+The resulting stability polynomial $P_{p;S}$ is then used to construct an optimized Runge-Kutta method.
+Trixi.jl implements the [Paired-Explicit Runge-Kutta (PERK)](https://doi.org/10.1016/j.jcp.2019.05.014) method, a low-storage, multirate-ready method with optimized domain of absolute stability.
+
+### Paired-Explicit Runge-Kutta (PERK) Schemes
+
+Paired-Explicit Runge-Kutta (PERK) or `PairedExplicitRK` schemes are an advanced class of numerical methods designed to efficiently solve ODEs.
+In the [original publication](https://doi.org/10.1016/j.jcp.2019.05.014), second-order schemes were introduced, which have been extended to [third](https://doi.org/10.1016/j.jcp.2022.111470)- and [fourth](https://doi.org/10.48550/arXiv.2408.05470)-order in subsequent works.
+
+By construction, PERK schemes are suited for integrating multirate systems, i.e., systems with varying characteristics speeds throughout the domain.
+Nevertheless, due to their optimized stability properties and low-storage nature, the PERK schemes are also highly efficient when applied as standalone methods. In Trixi.jl, the standalone PERK integrators require that all stages of the method be active.
+
+#### Tutorial: Using `PairedExplicitRK2`
+
+In this tutorial, we will demonstrate how you can use the second-order PERK time integrator. You need the packages `Convex.jl` and `ECOS.jl`, so be sure they are added to your environment.
+
+1. First, you need to load the necessary packages:
+
+```@example PERK-example-1
+using Convex, ECOS
+using OrdinaryDiffEq
+using Trixi
+```
+
+2. Define the ODE problem and the semidiscretization setup. For this example, we will use a simple advection problem.
+
+```@example PERK-example-1
+# Define the mesh
+cells_per_dimension = 100
+coordinates_min = 0.0
+coordinates_max = 1.0
+mesh = StructuredMesh(cells_per_dimension, 
+                      coordinates_min, coordinates_max)
+
+# Define the equation and initial condition
+advection_velocity = 1.0
+equations = LinearScalarAdvectionEquation1D(advection_velocity)
+
+initial_condition = initial_condition_convergence_test
+
+# Define the solver
+solver = DGSEM(polydeg = 3, surface_flux = flux_lax_friedrichs)
+
+# Define the semidiscretization
+semi = SemidiscretizationHyperbolic(mesh, 
+                                    equations, initial_condition, 
+                                    solver)
+```
+
+3. Define the necessary [callbacks](callbacks.md) for the simulation. Callbacks are used to perform actions at specific points during the integration process.
```

Comment:
```suggestion
3. Define the necessary [callbacks](@ref callbacks-id) for the simulation. Callbacks are used to perform actions at specific points during the integration process.
```
Does the link work as you intend? I would have expected something using `@ref`, which uses the [identifier](https://github.com/trixi-framework/Trixi.jl/blob/897c1cdf693321aa4b52e9737381661223d65645/docs/src/callbacks.md#L1C19-L1C31) of the caption.

Reply:
It does, but your suggestion is more uniform to the way the rest of the things in the documentation is linked. Thx!

Reply:
Interesting that linking to the markdown file itself also works. I didn't know that.